client/webserver/site/src/js/settings.js

```diff
@@ -124,6 +139,28 @@ export default class SettingsPage extends BasePage {
     Doc.hide(page.forms)
   }
 
+  // disableAccount disables the account associated with the provided host.
+  async disableAccount () {
+    const page = this.page
+    const pw = page.disableAccountAppPW.value
+    const host = page.disableAccountHost.textContent
+    page.disableAccountAppPW.value = ''
+    const req = {
+      pw,
+      host
+    }
+    const loaded = app.loading(this.body)
+    const res = await postJSON('/api/disableaccount', req)
+    loaded()
+    if (!app.checkResponse(res)) {
+      page.disableAccountErr.textContet = res.msg
+      Doc.show(page.disableAccountErr)
+    }
+    Doc.hide(page.forms)
```

Comment:
A few issues here.

1. If you're going to display the error on the form, should should pass `true` for the second argument of `checkResponse`. That argument is called `skipNote`, and it prevents the error from going to the notification bell.
2. You are not actually populating the text of `disableAccountErr`, since you have a typo in the property name. `textContet` should be `textContent`
3. Even if you were populating `disableAccountErr` correctly, you are immediately hiding the form anyway, so nobody will ever see it. You'll need to `return` after `Doc.show(page.disableAccountErr)`.